app/helpers/application_helper.rb

```diff
@@ -1,10 +1,22 @@
 module ApplicationHelper
   def flash_class(level)
     case level.to_sym
-      when :notice then "alert alert-info"
-      when :success then "alert alert-success"
-      when :error then "alert alert-danger"
-      when :alert then "alert alert-danger"
+    when :notice then "alert alert-info"
+    when :success then "alert alert-success"
+    when :error then "alert alert-danger"
+    when :alert then "alert alert-danger"
+    end
+  end
+
+  def title
+    if content_for?(:title)
+      content_for :title
```

Comment:
Looks like there is no case where we need to override the page title, so you can remove this conditional now